src/jobs.rs

```diff
@@ -361,6 +361,18 @@ impl CommitJobInstance {
 
         let mut commits = HashMap::new();
 
+        let mut endoflife_rebase_arg = String::new();
+        if let Some(endoflife_rebase) = &self.endoflife_rebase {
+            for build_ref in build_refs.iter() {
+                if build_ref.ref_name.starts_with("app/") {
+                    let parts: Vec<&str> = build_ref.ref_name.split('/').collect();
+                    let app_id = &parts[1];
+
+                    endoflife_rebase_arg = format!("--end-of-life-rebase={}={}", app_id, endoflife_rebase);
+                };
+            };
```

Comment:
Also this could be something like:
```
if let Some(first_ref) = build_refs.iter().filter(|ref| build_ref.ref_name.starts_with("app/")).nth(0) {
  endoflife_rebase_arg = format!("--end-of-life-rebase={}={}", app_id first_ref.ref_name.split('/').nth(1).unwrap();
}
```